src/main/java/org/fundacionjala/coding/lucero/Kata.java

```diff
@@ -11,6 +13,26 @@
     private Kata() {
     }
 
+    public static int countMostFrequentItem(final int[] data) {
```

Comment:
Please consider moving this utility method to another class, what is sought with this is that the solution for each exercise has its own classes (one for implementation and other for unit testing).